src/System.Buffers/src/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <PackageConfigurations>
       netstandard1.1;
       netstandard;
+      netfx;
```

Comment:
nit: netfx is ok in 2.1, but please make sure to use the versioned targetgroup in other branches.

Reply:
Will do.